contracts/EverlongStrategy.sol

```diff
@@ -0,0 +1,482 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.22;
+
+import { IHyperdrive } from "hyperdrive/contracts/src/interfaces/IHyperdrive.sol";
+import { FixedPointMath } from "hyperdrive/contracts/src/libraries/FixedPointMath.sol";
+import { SafeCast } from "hyperdrive/contracts/src/libraries/SafeCast.sol";
+import { HyperdriveUtils } from "hyperdrive/test/utils/HyperdriveUtils.sol";
```

Comment:
How is this being used? We don't need to change this now, but I don't love the idea of using test code in production. We should move the logic that we need into a periphery contract.

Reply:
It's being used for `calculateMaxLong()` to determine the `availableDepositLimit()`, nothing mission critical but agreed on moving the logic into a periphery contract... will probably throw it in `HyperdriveExecution` lib